cache/secondary_cache_adapter.cc

```diff
@@ -241,9 +241,10 @@ Status CacheWithSecondaryAdapter::Insert(const Slice& key, ObjectPtr value,
   }
   // Warm up the secondary cache with the compressed block. The secondary
   // cache may choose to ignore it based on the admission policy.
-  if (value != nullptr && !compressed_value.empty()) {
+  if (value != nullptr && !compressed_value.empty() &&
+      adm_policy_ == TieredAdmissionPolicy::kAdmPolicyThreeQueue) {
```

Comment:
Just for my understanding. For kAdmPolicyAuto, it won't insert?

Reply:
Right. This is restricted specifically to the primary + compressed secondary + nvm cache combination.